Bond/Bond+UIButton.swift

```diff
@@ -27,13 +27,14 @@
 
 import UIKit
 
-@objc class ButtonDynamicHelper
+class ButtonDynamicHelper: NSObject
```

Comment:
Sure, no worries — I wasn't suggesting that we change any of the "real" Swift class names — just what's exposed to Objective-C. It would be good to consider that for the big update you have coming :smile: